### PR TITLE
Add support levels

### PR DESCRIPTION
**Pull request by samthor**
Resolves [#43](https://github.com/web-animations/web-animations-js/issues/43).

### CODE REVIEW

**Comment by samthor** [comment]
@alancutter @dstockwell

File: README.md [hunk 1]
```diff
@@ -20,9 +20,14 @@ For more details see the
 What is the polyfill?
 ---------------------
 
-The polyfill is a JavaScript implementation of the Web Animations API. It works
-on modern versions of all major browsers. For more details about browser
-support see <https://www.polymer-project.org/resources/compatibility.html>.
+The polyfill is a JavaScript implementation of the Web Animations API. It is
+supported on modern versions of all major browsers, including-
```
Comment (alancutter) [inline README.md]: The - after "including" is a bit odd, did you mean "including:"?

**Comment by alancutter** [comment]
lgtm

File: README.md (hunk 1 above)
Comment (samthor) [inline README.md]: Done

**Comment by clemens** [comment]
Sorry to comment on this closed PR but I have a follow-up question: What prevents this library from supporting older versions of Safari? I'm asking because it's potentially relevant to a project I'm working on and in my experience with lots of modern browser features (including W3C drafts and recommendations) work just fine starting with Safari 5 (wich would inherently mean support for Snow Leopard users – who don't have a huge market share but they still exists).

**Pull request merged**